package.json

```diff
@@ -30,7 +30,7 @@
     "tailwindcss": "^1.4.0"
   },
   "dependencies": {
-    "bindings": "^1.5.0",
+    "@storybook/addon-knobs": "^6.1.6",
```

Comment:
seems like you were not on the latest version when you branched off the branch `update-storybook`? you can try 
```
git checkout update-storybook
git pull
git checkout dropdown
git rebase update-storybook
```